src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestV4ServiceImplTest.java

```diff
@@ -0,0 +1,754 @@
+import java.util.stream.Stream;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.MutableDateTime;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.kuali.kfs.core.api.config.property.ConfigurationService;
+import org.kuali.kfs.core.api.datetime.DateTimeService;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.kfs.sys.KFSConstants;
+import org.mockito.Mockito;
+
+import edu.cornell.kfs.concur.ConcurConstants.ConcurEventNoticationVersion2EventType;
+import edu.cornell.kfs.concur.ConcurConstants.RequestV4Status;
+import edu.cornell.kfs.concur.ConcurKeyConstants;
+import edu.cornell.kfs.concur.ConcurParameterConstants;
+import edu.cornell.kfs.concur.ConcurTestConstants;
+import edu.cornell.kfs.concur.ConcurTestConstants.ParameterTestValues;
+import edu.cornell.kfs.concur.ConcurTestConstants.PropertyTestValues;
+import edu.cornell.kfs.concur.ConcurTestConstants.RequestV4Dates;
+import edu.cornell.kfs.concur.batch.fixture.ConcurFixtureUtils;
+import edu.cornell.kfs.concur.batch.service.ConcurBatchUtilityService;
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationV2WebserviceService;
+import edu.cornell.kfs.concur.batch.service.impl.fixture.RequestV4DetailFixture;
+import edu.cornell.kfs.concur.batch.service.impl.fixture.RequestV4ListingFixture;
+import edu.cornell.kfs.concur.batch.service.impl.fixture.RequestV4PersonFixture;
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotificationProcessingResultsDTO;
+import edu.cornell.kfs.concur.businessobjects.ValidationResult;
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurRequestV4CustomItemDTO;
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurRequestV4ListItemDTO;
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurRequestV4ListingDTO;
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurRequestV4OperationDTO;
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurRequestV4PersonDTO;
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurRequestV4ReportDTO;
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurRequestV4StatusDTO;
+import edu.cornell.kfs.concur.service.ConcurAccountValidationService;
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.service.impl.TestDateTimeServiceImpl;
+import edu.cornell.kfs.sys.web.mock.MockRemoteServerExtension;
+
+@Execution(ExecutionMode.SAME_THREAD)
```

Comment:
I am able to run this unit test class without this execution statement.  Is it still needed?

Reply:
I was concerned about whether the tests would encounter problems if parallel execution was enabled, but I agree that they seem to run fine locally with such a setup. I'll remove the annotation and see if Jenkins handles the parallel test runs properly.